client/src/component/Footer.jsx

```diff
@@ -1,5 +1,6 @@
 import React from 'react';
-
+import 'font-awesome/css/font-awesome.min.css'
+import './footer.css'
 function Footer() {
```

Comment:
Change to a class, because we will be working with states later on
```suggestion
class Welcome extends React.Component {
  render() {
    return <h1>Hello, {this.props.name}</h1>;
  }
}
```

Reply:
I would say leave this as a function, but turn this to an arrow function. @adriandarian, I don't think a footer would ever have a state

Reply:
@shiv248 you can change this to an arrow function because there are no state changes happening here